resources_pages/install_ds_stack_windows.md

```diff
@@ -3,290 +3,540 @@ layout: page
+To see if you were successful, try opening RStudio by clicking on its icon. It should open and looks something like this picture below:
 
-- Python (everything Python: notebooks, debugging, linting, formatting, etc.)
-- markdownlint (markdown linting and style checking extension)
-- GitLens (powerful extension that extends VS Code's native git capabilities)
-- Docker (easily use Docker from VS Code)
-- (Optional) Material Theme and/or Predawn Theme Kit (additional colour themes to choose from)
-- (Optional) Material Icon Theme (great-looking custom file icons!)
-- (Optional) Bracket Pair Colorizer 2 (add colour to help distinguish your brackets: (), [], {})
+![](/resources_pages/imgs/RStudio.png)
 
-This [video tutorial](https://www.youtube.com/watch?v=-nh9rCzPJ20) is an excellent introduction to using VS Code in Python.
+By default, RStudio installs packages to `C:/Users/your_username/Documents/R/win-library/4.0`, but when it is run directly from the terminal it instead uses  `C:/Users/your_username/R/win-library/4.0`. To make sure that R from terminal uses the same package directory as R studio, we will edit `~/.Rprofile` by typing:
+
+```
+code ~/.Rprofile
+```
+
+And adding the following
+
+```
+# Make R from terminal use the same package directory as RStudio
+.libPaths(c(paste("C:/Users/", Sys.getenv("USERNAME"), "/Documents/R/win-library/4.0" , sep=''), .libPaths()))
+```
+
+### Essential R packages
+
+Next, install the key R packages needed for the start of MDS program,
+by typing the following into the R terminal in RStudio:
+
+```
+install.packages(c('tidyverse', 'blogdown', 'xaringan', 'renv'))
+```
+
+If you get a prompt asking if you want to install packages that need compilation from sources, click "Yes".
+
+> Note: we will use many more packages than those listed above across the MDS program, however we will manage these using the `renv` package manager (which you will learn about in DSCI 521: Platforms for Data Science).
+
+### Rtools
+
+Windows users will also need to install Rtools, which will allow you to use external libraries. Go to <http://cran.r-project.org/bin/windows/Rtools/> and download the latest version (e.g., Rtools40.exe). After the download has finished, run the installer with the default configuration. After the installation has completed, follow the steps on the Rtools page under the heading "Putting Rtools on the PATH".
+
+To use Rtools from R in terminal, edit `~/.Rprofile`:
+
+```
+code ~/.Rprofile
+```
+
+and append the following:
+
+```
+# Enable Rtools from R in the terminal
+Sys.setenv(PATH = paste("C:/rtools40/usr/bin/", Sys.getenv("PATH"), sep=";"))
+```
+
+### IR kernel
+
+The `IRkernel` package is needed to make R work in Jupyter notebooks. To enable this kernel in the notebooks, open R *from a terminal* and run the setup via the following two commands:
```

Comment:
I tried for a few hours to get RStudio to see my jupyter installation but no luck. Even when I added it to path the same way as I added rtools above, it would say it found Jupyter but could not run it. Launching from Git Bash sets the R PATH to the path setu in gitbash which contains all miniconda executables. This is now similar to the macOS instructions which also uses R in terminal for this step.